app/code/Magento/MediaGalleryUi/Model/SearchCriteria/CollectionProcessor/FilterProcessor/Keyword.php

```diff
@@ -57,6 +57,7 @@ public function apply(Filter $filter, AbstractDb $collection): bool
      */
     private function getAssetIdsByKeyword(string $value): array
     {
+        $keywordValues = $this->splitRequestString($value);
```

Comment:
I'd avoid one time use variables, the call to `splitRequestString ` can be done on line 71